packages/web-twig/src/Resources/components/Header/stories/HeaderMinimal.twig

```diff
@@ -0,0 +1,3 @@
+<Header>
+    {% include '@components/Header/stories/_partials/JobBoardLogo.twig' %}
```

Comment:
The implicit task is to implement ProductLogo in Twig? :D

Reply:
No, we no longer create new components in Twig 🙂.